modules/compiler/spirv-ll/test/glsl/builtin_var_gl_GlobalInvocationID.comp

```diff
@@ -26,8 +26,8 @@ void main() {
 
 // CHECK: ; ModuleID = '{{.*}}'
 // CHECK: define spir_kernel void @main() #0 {
-// CHECK: {{[%@][-a-zA-Z$._0-9][-a-zA-Z$._0-9]*}} = alloca i32
-// CHECK: {{[%@][-a-zA-Z$._0-9][-a-zA-Z$._0-9]*}} = call spir_func {{i[0-9]+}} @_Z13get_global_idj(i32 0)
-// CHECK: store i32 {{[%@][-a-zA-Z$._0-9][-a-zA-Z$._0-9]*}}, ptr {{[%@][-a-zA-Z$._0-9][-a-zA-Z$._0-9]*}}
+// CHECK: {{[%@].*}} = alloca i32
+// CHECK: {{[%@].*}} = call spir_func {{i[0-9]+}} @_Z13get_global_idj(i32 0)
```

Comment:
Even further reduced:
```suggestion
// CHECK: = alloca i32
// CHECK: = call spir_func {{i[0-9]+}} @_Z13get_global_idj(i32 0)
```
and likewise for the others? This is what I can find in plenty of LLVM tests.

Reply:
Interesting idea, yeah. Let me get out my `sed`.

Reply:
Done!